### PR TITLE
chore(fulfillment): drop mockend

### PR DESCRIPTION
**Pull request by Sinucid**
closes: [HRZ-90200](https://spryker.atlassian.net/browse/HRZ-90200)

[HRZ-90200]: https://spryker.atlassian.net/browse/HRZ-90200?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: apps/fulfillment/.env.template [hunk 1]
```diff
@@ -1,3 +1,2 @@
-ORYX_FULFILLMENT_BACKEND_URL=http://localhost:3001
 ORYX_LABS=true
```
Comment (tobi-or-not-tobi) [inline apps/fulfillment/.env.template]: I'd drop labs from the template, we don't use it and dont' have anything in labs for picking afaik.

**Review by tobi-or-not-tobi — DISMISSED** [review]

File: libs/template/presets/fulfillment/app.ts [hunk 2]
```diff
@@ -24,6 +27,16 @@ export function fulfillmentFeatures(
...
+    siteFeature,
+    formFeature,
+    {
+      //drop PageMetaResolver from experienceFeature
+      //to exclude unnecessary functionality from SPA
+      providers: experienceFeature.providers?.filter(
+        (feature) => ![PageMetaResolver].includes(feature.provide)
+      ),
+      components: experienceFeature.components,
+    },
```
Comment (supproduction) [inline libs/template/presets/fulfillment/app.ts:39]: Is it still relevant?

**Review by supproduction — DISMISSED** [review]

File: libs/template/presets/fulfillment/app.ts (hunk 2 above)
Comment (Sinucid) [reply libs/template/presets/fulfillment/app.ts:39]: Due to FA still not use static experience data for its pages (components instead) and PageMetaResolver is not satisfied with this

Comment (tobi-or-not-tobi) [reply libs/template/presets/fulfillment/app.ts:39]: also we'll use it shortly for the work Alexander is doing

**Review by tobi-or-not-tobi — APPROVED** [review]

**Pull request merged**